pkg/ipamd/ipamd.go

```diff
@@ -368,25 +383,26 @@ func New(rawK8SClient client.Client, cachedK8SClient client.Client) (*IPAMContex
 		return nil, err
 	}
 
-	hypervisorType := c.awsClient.GetInstanceHypervisorFamily()
-
-	if hypervisorType != "nitro" && c.enableIpv4PrefixDelegation {
-		log.Warnf("Prefix delegation is not supported on non-nitro instance %s hence falling back to default (secondary IP) mode", c.awsClient.GetInstanceType())
-		c.enableIpv4PrefixDelegation = false
+	//Let's validate if the configured combination of env variables is supported before we
+	//proceed any further
+	if !c.isConfigValid() {
```

Comment:
We might either have to set `ENABLE_IPv4` -> `true` by default or generate two manifests for v4 and v6 with `ENABLE_IPv4=true` in v4 manifest and `ENABLE_IPv6=true` in v6 manifest. Since if anyone builds based of master then aws-node will fail to start.

Reply:
We will include the new env vars in the default manifest.

Reply:
Agreed but this will fail nightly integration test :(

Reply:
So the default manifest will have v4 set to true?

Reply:
Yeah, v4 will be enabled by default.